host/sgx/sgxquote.c

```diff
@@ -5,14 +5,25 @@
 #include "sgxquote.h"
 #include <openenclave/internal/defs.h>
 #include <openenclave/internal/raise.h>
+#include <openenclave/internal/sgx/plugin.h>
+#include <openenclave/internal/trace.h>
 #include <sgx_dcap_ql_wrapper.h>
+#include <string.h>
 
 // Check consistency with OE definition.
 OE_STATIC_ASSERT(sizeof(sgx_target_info_t) == 512);
 OE_STATIC_ASSERT(sizeof(sgx_report_t) == 432);
 
-oe_result_t oe_sgx_qe_get_target_info(uint8_t* target_info)
+oe_result_t oe_sgx_qe_get_target_info(
+    const oe_uuid_t* format_id,
+    const void* opt_params,
+    size_t opt_params_size,
+    uint8_t* target_info)
 {
+    OE_UNUSED(format_id);
+    OE_UNUSED(opt_params);
+    OE_UNUSED(opt_params_size);
```

Comment:
Why are the 3 parameters added but not used?

Reply:
These parameters will be used when the quote-ex library is actually added (which would be a separate PR).